### PR TITLE
new AMI for tesla analysis

### PR DESCRIPTION
**Pull request by xdoan**

### CODE REVIEW

File: config/prod/tesla-analysis.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
+stack_name: tesla-analysis
```
Comment (zaro0508) [inline config/prod/tesla-analysis.yaml]: would you mind adding `ec2` the this name?  and also change the filename to match?  something like `tesla-analysis-ec2` would work.

File: config/prod/tesla-analysis.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+parameters:
+  # The Sage deparment for this resource
+  Department: "CompOnc"
+  # The Sage project this resource will be used for
+  Project: "TESLA"
+  # The resource owner
+  OwnerEmail: "xengie.doan@sagebase.org"
+  # EC2 instance type (available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "m5.2xlarge"
```
Comment (zaro0508) [inline config/prod/tesla-analysis.yaml]: wondering why you chose m5 instance?  why not t3 or t2?

Comment (xdoan) [reply config/prod/tesla-analysis.yaml]: Tom and I were browsing through resources and STAR aligner recommended 100G disk/30G RAM/6+ CPUs. I would be happy to change it if t3 or t2 works better.

Comment (xdoan) [reply config/prod/tesla-analysis.yaml]: Probably do not need that much disk space if I run through batches of files and delete them, but anything with that much memory and CPUs is good.

Comment (zaro0508) [reply config/prod/tesla-analysis.yaml]: all have the same specs[1],  16 GB ram and 4 vCPU.  if you want something larger then i would recommend t3.2xlarge ?  AWS says all should work the same except t3 is the least expensive option.

[1] https://aws.amazon.com/ec2/instance-types/

Comment (xdoan) [reply config/prod/tesla-analysis.yaml]: Sounds good, I changed it to t3.2xlarge.

**Pull request merged**